README.md

```diff
@@ -71,6 +71,7 @@ From there:
 
 - [Arch Linux](#arch-linux)
 - [Snap](#snap)
+- [Nix](#nix)
```

Comment:
Let's to alphabetical order here and bellow as then distro wars can't happen about the order.